omnibus/package-scripts/agent/posttrans

```diff
@@ -55,6 +55,31 @@ else
     echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd and upstart."
 fi
 
+# SELinux policy not shipped on SUSE for now
+if [ "$DISTRIBUTION_FAMILY" != "SUSE" ]; then
+    # Setup SELinux policy and label if SELinux detected on the host
+    if command -v semodule >/dev/null 2>&1 && [ -f "$INSTALL_DIR/embedded/bin/system-probe" ]; then
+        echo "Loading SELinux policy module for datadog-agent."
+        semodule -v -i $CONFIG_DIR/selinux/system_probe_policy.pp >/dev/null 2>&1
+        if [ "$?" != "0" ]; then
+            echo "The SELinux policy module for datadog-agent is not compatible with this host."
+            echo "Please install or update the selinux-policy-targeted package and then reinstall the datadog-agent package to configure SELinux for the Datadog Agent."
+            echo "Skipping SELinux configuration."
+        else
+            echo "Labeling SELinux type for the system-probe binary."
+            if command -v semanage >/dev/null 2>&1 && command -v restorecon >/dev/null 2>&1;then
+                semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe
+                restorecon -v $INSTALL_DIR/embedded/bin/system-probe
+            else
+                echo "Could not set up SELinux labels for datadog-agent: semanage and/or restorecon commands not found."
+                echo "Please make sure they are installed (via the policycoreutils, policycoreutils-python and/or policycoreutils-python-utils packages), and then run:"
```

Comment:
should these packages be added as weak or very weak dependencies for the `datadog-agent` RPM?

Reply:
The main problem is that it depends on the version of the target distribution: CentOS/RHEL 7 has `policycoreutils-python`, and CentOS/RHEL 8 has `policycoreutils-python-utils`, and I don't know how the package manager is going to behave in the case where a dependency does not exist.